catalog.html

```diff
@@ -224,25 +226,34 @@ <h2>Напишите нам</h2>
                     <textarea id="message" class="txtarea" name="message" placeholder="В свободной форме:"></textarea>
               <button id="send" type="submit">Отправить</button>
                 </form>
-        </section>
-    </main>
+          </section>
+  
+      </main>
 
-    <footer><!--Подвал-->
+    <footer>
         <section>
           <ul>
             <li>
-              <a href="#">Вконтакте</a>
+              <a href="https://vk.com/">
+                <img src="img/vk-icon.png"  width="26" height="15" alt="vk">
```

Comment:
```suggestion
                <img src="img/vk-icon.svg"  width="26" height="15" alt="vk">
```
Иконки соц. сетей должны быть в формате SVG,